web_src/css/repo.css

```diff
@@ -655,9 +655,19 @@
   .repository.view.issue .issue-title {
     flex-direction: column;
   }
+  .repository.view.issue .issue-title-buttons, .repository.view.issue .edit-buttons {
+    width: 100%;
+    justify-content: space-between;
+  }
   .repository.view.issue .edit-buttons {
     margin-top: .5rem;
   }
+  .repository.new.issue .comment.form .comment .avatar, .repository.pull .issue-comment .comment .avatar {
```

Comment:
This selector is so long (and others, above/below), I guess it doesn't need to be this long?